staging/src/k8s.io/client-go/rest/watch/decoder_test.go

```diff
@@ -70,7 +70,7 @@ func TestDecoder(t *testing.T) {
 		go func() {
 			action, got, err := decoder.Decode()
 			if err != nil {
-				t.Fatalf("Unexpected error %v", err)
+				t.Errorf("Unexpected error %v", err)
```

Comment:
would at least make sure that this go func returns early.

Reply:
Also probably for all other changes: make sure that the code does not panic after the error.

Reply:
@sttts  thanks for reply.
does https://github.com/kubernetes/kubernetes/commit/fca4ef700710001f0e2e73f553a93f56fde60aa0 work?

Reply:
@sttts  thanks for reply.  How about the new fix?

Reply:
kindly ping for long time stuck  @sttts